contrib/ros/README.md

```diff
@@ -0,0 +1,138 @@
+| Sawyer       | Ubuntu16.04    | Kinetic| 3      |Gazebo 7.0           |
+## Installation
+- Assuming we got a clean system which only has rllab running.
+- Check if our robots and workstations are supported by rllab.contrib.ros    
+- Following is how we setup the rllab.contrib.ros's environment for sawyer robot
+### Setup Workstation
+#### INSTALL ROS
+##### Configure Ubuntu repositories
+Configure your Ubuntu repositories to allow "restricted," "universe," and "multiverse." **Likely, they are already configured properly, and you only need to confirm the configuration.**
+##### Setup your sources.list
+    $ sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu xenial main" > /etc/apt/sources.list.d/ros-latest.list'
+##### Setup your keys
+    $ sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-key 421C365BD9FF1F717815A3895523BAEEB01FA116
+##### Update to Latest Software Lists
+    $ sudo apt-get update
+##### Install ROS Kinetic Desktop Full
+    $ sudo apt-get install ros-kinetic-desktop-full
+##### Initialize rosdep
+rosdep enables you to easily install system dependencies for source you want to compile and is required to run some core components in ROS.
+    
+    $ sudo rosdep init
+    $ rosdep update
+##### Install rosinstall
+    $ sudo apt-get install python-rosinstall
+#### CREATE DEVELOPMENT WORKSPACE
+##### Create ROS Workspace
+    $ mkdir -p ~/<YOUR_FAVORITE_WORKSPACE_NAME>/src
+##### Source ROS Setup
+    $ source /opt/ros/kinetic/setup.bash
+You can also add this to you .bashrc by using
+    
+    $ echo "source /opt/ros/kinetic/setup.bash" >> ~/.bashrc
+##### Build
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ catkin_make
+#### INSTALL INTERA SDK DEPENDENCIES
+##### Install SDK Dependencies
+    $ sudo apt-get update
+    $ sudo apt-get install git-core python-argparse python-wstool python-vcstools python-rosdep ros-kinetic-control-msgs ros-kinetic-joystick-drivers ros-kinetic-xacro ros-kinetic-tf2-ros ros-kinetic-rviz ros-kinetic-cv-bridge ros-kinetic-actionlib ros-kinetic-actionlib-msgs ros-kinetic-dynamic-reconfigure ros-kinetic-trajectory-msgs ros-kinetic-rospy-message-converter
+#### INSTALL INTERA ROBOT SDK
+##### Activate the conda environment for rllab
+    $ source activate rllab3
+
+**rllab3 is your rllab conda environment's name** 
+##### Download the SDK on your Workstation
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>/src
+    $ wstool init .
+    $ git clone https://github.com/RethinkRobotics/sawyer_robot.git
+    $ wstool merge sawyer_robot/sawyer_robot.rosinstall
+    $ wstool update
+##### Download the dependecies 
+    $ git clone https://github.com/ros/geometry.git
+    $ cd geometry
+    $ git checkout indigo-devel
+##### Source ROS Setup
+    $ source /opt/ros/kinetic/setup.bash
+##### Build
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ catkin_make -DPYTHON_EXECUTABLE:FILEPATH=<path_to_your_rllab_env_python_interp>
```

Comment:
Can you provide more info to the user on how to locate `<path_to_your_rllab_env_python_interp>`? They are likely not experts on conda. You can probably extract it into a variable to help them.

```sh
cd $YOUR_ROS_WS
export RLLAB_PYTHON=`conda env rllab3 --something`
catkin_make -DPYTHON_EXECUTABLE:FILEPATH=$RLLAB_PYTHON
```

Reply:
Ex. "/home/sawyer/anaconda2/envs/rllab3/bin/python"

Reply:
What I used:
```bash
$ source activate rllab3
$ export RLLAB_PYTHON=`which python`
$ source deactivate
```